test/e2e/operator_test.go

```diff
@@ -147,7 +147,9 @@ var _ = Describe("Operator API", func() {
 
 		// Ensure o's status.components.refs field eventually contains a reference to ns-a
 		By("eventually listing a single component reference")
-		componentRefEventuallyExists(w, true, getReference(scheme, nsA))
+		ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
```

Comment:
I think this should be using `clientCtx` instead of `context.Background()`.

Reply:
I change them.  Thanks!